src/components/sbb-option/sbb-option.stories.jsx

```diff
@@ -29,21 +44,44 @@ const numberOfOptions = {
 const defaultArgTypes = {
   value,
   'icon-name': iconName,
-  active: active,
+  active,
+  disabled,
   numberOfOptions,
+  preserveIconSpace,
 };
 
 const defaultArgs = {
   value: 'Value',
   'icon-name': undefined,
   active: false,
+  disabled: false,
   numberOfOptions: 5,
+  preserveIconSpace: false,
 };
 
-const createOptions = ({ value, numberOfOptions, ...args }) =>
-  new Array(numberOfOptions).fill(null).map((_, i) => {
-    return <sbb-option {...args} value={`${value} ${i + 1}`}>{`${value} ${i + 1}`}</sbb-option>;
+const createOptions = ({
+  value,
+  active,
+  disabled,
+  numberOfOptions,
+  preserveIconSpace,
+  ...args
+}) => {
+  const style = preserveIconSpace ? '--sbb-option-icon-container-display: block;' : '';
```

Comment:
Little bit confused, should this not be a property of sbb-option? or at list a dataset entry of sbb-option? Maybe I don't understand it correctly, but seems like a very hidden feature :D